packages/core/src/components/Autocomplete/Autocomplete.example.jsx

```diff
@@ -0,0 +1,64 @@
+import Autocomplete from './Autocomplete';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TextField from '../TextField/TextField';
+
+ReactDOM.render(
+  <div>
+    <Autocomplete
+      label="Select from the options below:"
+      itemToString={i => {
+        return i ? i.name : '';
+      }}
+      items={[
+        {
+          id: 'kRf6c2fY',
+          name: 'Cook County, IL'
+        },
+        {
+          id: 'lYf5cGfM',
+          name: 'Cook County, MD'
+        },
+        {
+          id: 'mZfKcGf9',
+          name: 'Cook County, TN'
+        }
+      ]}
+      onChange={selectedItem => console.log(selectedItem)}
+    >
+      <TextField
+        hint="This is an autocomplete field. Begin typing to search for relevant information. The number of results will be updated as you type."
+        label="Constrained list"
```

Comment:
We don't explain anywhere what "Constrained" or "Unconstrained" means, so we could rephrase these field labels to something maybe more clear, like: "Labeled list" and "Simple list".

Reply:
Done. Labels changed to suggested "Labeled" and "Simple" lists.